src/STW.StubApi.Presentation/Services/Notification/Helpers/NotificationHelper.cs

```diff
@@ -0,0 +1,13 @@
+namespace STW.StubApi.Presentation.Services.Notification.Helpers;
+
+public static class NotificationHelper
+{
+    public static string GenerateReferenceNumber()
+    {
+        var currentYear = DateTime.Now.Year;
+        var rand = new Random();
+        var random = "1" + rand.Next(0, 1000000).ToString("D6");
+
+        return $"DRAFT.GB.{currentYear}.{random}";
```

Comment:
Should be submitted not draft I think, at this point we'd expect IPAFFS to do whatever is required to make it submitted, and that's the reference we give back to STW. So prefix would be certificate type